{"name": "Курьер не может авторизоваться под несуществующим пользователем", "status": "failed", "statusDetails": {"message": "AssertionError: assert (404 == 400)\n +  where 404 = <Response [404]>.status_code", "trace": "self = <tests.test_login_courier.TestLoginCourier object at 0x000002051584D310>\n\n    @allure.title(\"Курьер не может авторизоваться под несуществующим пользователем\")\n    def test_courier_cannot_log_in_nonexistent_user(self):\n        payload = {'login': 'Saske', 'password': '5555'}\n        response = requests.post(f\"{Url.PAGE_SCOOTER}{Url.LOGIN_COURIER}\", data=payload)\n>       assert response.status_code == 400 and LoginCourierAnswer.answer_code_400 in response.text\nE       assert (404 == 400)\nE        +  where 404 = <Response [404]>.status_code\n\ntests\\test_login_courier.py:49: AssertionError"}, "start": 1728248355356, "stop": 1728248355515, "uuid": "9085ed46-9309-464d-935a-8a20e94de315", "historyId": "6a3d2435021227a0ed13cee24ed73372", "testCaseId": "6a3d2435021227a0ed13cee24ed73372", "fullName": "tests.test_login_courier.TestLoginCourier#test_courier_cannot_log_in_nonexistent_user", "labels": [{"name": "suite", "value": "Логин курьера"}, {"name": "parentSuite", "value": "tests"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "DESKTOP-COFSJ1F"}, {"name": "thread", "value": "9116-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}